### ISSUE TITLE
Expand Cypress-Axe accessibility test coverage

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#9921]
## Description
We should test a broader range of pages to be more confident about our accessibility.  Foreground/background text contrast in particular should be found fairly easily, but we have a number of issues with that.

## Acceptance Criteria
- [ ] Cypress accessibility tests have been expanded to cover other view modes for all content types.
- [ ] Other pages (content views, etc) are covered by accessibility tests.  Pretty much anything accessible by a `content_admin` user and used in the course of normal work.

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by ndouglas** [comment]
See #10364, which relates to the above complaint.  I shouldn't be manually retyping error messages.

**Comment by EWashb** [comment]
It looks like this one is ready to close if I'm reading the PR correctly, right @ndouglas ?

**Comment by ndouglas** [comment]
Nope; the tests can't pass until the accessibility issues are fixed.
